k8s/src/main/scala/io/buoyant/k8s/resources.scala

```diff
@@ -91,18 +91,18 @@ private[k8s] class NsVersion[O <: KubeObject](
 }
 
 /**
-  * A custom resource version contains CustomResource-typed objects. The API was called ThirdPartyResources in Kubernetes
-  * < 1.7 and has been deprecated since version 1.8. The APIs for both resource versions are identical in implementation. However,
-  * various naming standards have been changed to make the apis more meaningful. For more details
-  * see https://kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-third-party-resource/.
-  * To understand how to create your own custom resources see
-  * https://kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/ for a description of the
-  * design model.
-  *
-  * Implementors are likely to want to extend this to provide convenience methods on top of the `list` method.
-  *
-  * @tparam O The parent type for all [[KubeObject]]s served by this Version.
-  */
+ * A custom resource version contains CustomResource-typed objects. The API was called ThirdPartyResources in Kubernetes
+ * < 1.7 and has been deprecated since version 1.8. The APIs for both resource versions are identical in implementation. However,
+ * various naming standards have been changed to make the apis more meaningful. For more details
+ * see https://kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-third-party-resource/.
+ * To understand how to create your own custom resources see
+ * https://kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/ for a description of the
+ * design model.
+ *
+ * Implementors are likely to want to extend this to provide convenience methods on top of the `list` method.
+ *
+ * @tparam O The parent type for all [[KubeObject]]s served by this Version.
+ */
```

Comment:
same as above, unrelated formatting change. move this to a separate PR?

Reply:
This was probably my fault and happened when I added the CustomResourceDefinition changes, sorry 😨